ci/garbage_collection/cache_cleanups/cache_garbage_collection.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash 
+
+
+    
+. ../garbage_collection_misc.sh
+. ../prune_branches.sh
+
+time_limit=14     ## Days. Set this to give the "deadline". All
+                  ## branches older than this will be removed.
+
+cache_location_dir=/data2/openvdc-ci/branches
+
+
+###################################################################################
+
+## main()
+
+
+## Remove all directories whose branch (on git) no longer exists
+## or which has not beenm pushed to within $time_limit days.
+for directory in $(TIME_LIMIT=${time_limit} dirs_to_prune ${cache_location_dir}); do
+
+   group_owner=$(stat -c %G ${cache_location_dir}/${directory})
+   if [[ "${group_owner}" = "root" ]]; then
+       echo "${directory} is owned by root. Cannot remove!"
```

Comment:
The problem is that currently the acceptance test's cache is always owned by root. This code is ok like this but the multibox code should change the cache images group settings so this script can remove them.